src/Documentation/doc/advanced/corporation/faq.md

```diff
@@ -0,0 +1,211 @@
+
+> Everybody tells me to use 1/9 as Intern ratio, but when I use it, energy and morale still drop.
+
+You can only use that ratio when your corporation works fine (funds > 0 or profit > 0). If it does not, use 1/6.
+
+> Are there any other ways to maintain these 2 stats?
+
+There are researches for that. However, you should never buy them, it's always better spending your RP elsewhere or just stock up on RP.
+
+> Buying tea and throwing party cost me too much money. Why are they so expensive?
+
+Tea and party are cheap. If your budget is so low that they cost you too much money, it means you wasted too much funds.
+
+> How much money should I spend to throw party? How often should I buy tea / throw party?
+
+Check this [section](./office.md).
+
+> How do I know if the qualities of my input materials are too low and need to be improved?
+
+Check this [section](./quality.md).
+
+> What are Awareness and Popularity?
+
+Check this [section](./wilson-analytics-advert.md).
+
+> Should I buy Dream Sense?
+
+No. Check this [section](./wilson-analytics-advert.md) for the reason.
+
+> Is Wilson retroactive?
+
+No.
+
+> Does that mean I should buy Wilson as soon as possible? If yes, then why don't you buy any Wilson in round 1 and 2?
+
+You should buy Wilson as soon as possible, but not too soon. Round 1 and 2 are those cases. Check this [section](./wilson-analytics-advert.md) for details.
+
+> What are Demand and Competition?
+
+Check this [section](./demand-competition.md).
+
+> How much should I spend for "Design investment" and "Marketing investment" when I create new product? How do they affect the product?
+
+They are not too important. It's fine to spend 1% of your current funds for them. Check this [section](./product.md) for details.
+
+> Should I buy Market-TA1?
+
+No, wait for Market-TA2. Market-TA1 is useless.
+
+> When should I buy Market-TA2?
+
+As soon as possible, it greatly increases your profit because it can find the optimal price. However, that research is expensive, it costs totally 75000RP (Hi-Tech R&D Laboratory + Market-TA1 + Market-TA2). Depleting entire RP pool extremely degrades your product rating, so I recommend saving up 150000RP before buying it. Depleting half of the RP pool is acceptable, considering the positive effect of Market-TA2.
+
+> What is the difference between Market-TA1 and Market-TA2?
+
+Market-TA1: set a price that ensures that you can sell all produced goods in storage.  
+Market-TA2: set highest possible price that ensures that you can sell all produced goods in storage.
+
+> I bought Market-TA2, but it does not set optimal price for me.
+
+You have to enable it.
+
+> Is there a workaround for Market-TA2? Waiting time for RP is too long.
+
+Yes, you can reimplement Market-TA2. Implementing custom Market-TA2 script is the best optimization in round 3+. Check this [section](./optimal-selling-price-market-ta2.md) to see how to do it.
+
+> How do I discard material/product?
+
+Set selling price to 0.
```

Comment:
```suggestion
> How do I discard materials/products?

Set the selling price to 0 (or anything small enough).
```

Reply:
I'll remove "(or anything small enough)" in next commit. With material, low selling price boosts `MarkupMultiplier` if it's less than `MarketPrice`, so it's true that "small enough" price is fine if we only want to boost `MaxSalesVolume`. However, I want to emphasize the `0` when the goal is "discard all of them". When the selling price is 0, the game set `MarkupMultiplier` to $10^{12}$. This means the materials is discarded in next SALE state, regardless of other factors (business, awareness, popularity, quality, etc.).